team_red: ["jnogueira", "tmadeira", "talmeida","jmoreira"]
team_green: ["ttavares", "acastro","psilva","dsilva"]
team_blue: ["moliveira", "jsantos","drato","mferreira"]